packages/terra-site/src/examples/alert/InfoExample.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Alert from 'terra-alert';
+
+const alertStartTagStr = '<Alert type={Alert.Types.INFO} >';
+const alertEndTagStr = '</Alert>';
+const alertText = 'This is an information alert';
+
+const InfoExample = () => (
+  <div>
+    <div dir="ltr">
+      <h3>Basic Alert of type info with default title</h3>
+      <br />
+      <code>
```

Comment:
Do we want to expose the code implementation in our site examples? That isn't something we are doing anywhere else